src/language-html/utils/index.js

```diff
@@ -386,6 +386,10 @@ function inferStyleParser(node) {
   if (lang === "less") {
     return "less";
   }
+
+  if (lang === "stylus") {
+    return inferParserByLanguage("stylus", options);
+  }
```

Comment:
Can you add a comment that explains why `"stylus"` is here? e.g.:

```suggestion
  // Prettier does not officially support stylus.
  // But, we need to handle `"stylus"` here for printing a style block in Vue SFC as stylus code by external plugin.
  // https://github.com/prettier/prettier/pull/12707
  if (lang === "stylus") {
    return inferParserByLanguage("stylus", options);
  }
```

Reply:
Makes Sense. Done.